android/app/src/main/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "Email": "dev@fossasia.org",
   "App_Name": "Open Event",
-  "Api_Link": "https://eventyay.com/api/v1/events/6/"
+  "Api_Link": "https://api.eventyay.com/v1/events/69/"
```

Comment:
For testing we need sample data so that we can confirm app is working fine fetching response from server. When app is generated using api endpoint then this api_link will change and app will use that accordingly.